sources/functions/swizdb

```diff
@@ -49,6 +42,19 @@ _getswizdb() {
 }
 export -f _getswizdb
 
+#Recursively lists keys
+# \
+# $1 = optional directory to refine
+# \
+# **Returns error code of the command that fails**
+_listswizdb() {
+    key="$1"
+    echo_log_only "List of keys stored under directory \"$key\""
+    pushd "$dbdir/$key" > /dev/null || return $? # Return error code of pushd, but stderr is not redirected on purpose.
+    find * -type f || return $?
```

Comment:
I'm wondering if there's a better way to do this, but this is weirdly the simplest way to get what I want: just a line-separated list of files only without any prepending `./` or full path or anything like that

Reply:
```suggestion
    find "$dbdir/$key" -type f -exec basename {} \; | sort || return $?
```

Reply:
don't think we need to pushd in to the dir